bin/ch/Debugger.cpp

```diff
@@ -327,7 +327,7 @@ bool Debugger::SetBaseline()
                 script[numChars] = '\0';
 
                 JsValueRef wideScriptRef;
-                IfJsrtErrorFailLogAndRetFalse(ChakraRTInterface::JsCreateString(
+                IfJsErrorFailLog(ChakraRTInterface::JsCreateString(
```

Comment:
This change in the macro changes the return value of the function in this case; since hr will still be S_OK, the function will now return true (where previously it was returning false). It might be more appropriate here to either make a macro that allows assignment to a variable in the error case (so you could set hr = E_FAIL) before jumping to Error.